packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2295,10 +2295,18 @@ abstract class BuildContext {
   /// data down to them.
   void visitChildElements(ElementVisitor visitor);
 
-  /// Returns a description of an [Element] from the current build context.
+  /// Returns a description of the [Element] associated with the current build context.
+  ///
+  /// The `name` is typically something like "The element being rebuilt was".
```

Comment:
nit (and not really related to this PR): `name` seems like an unfortunate naming choice for a parameter that contains a string like that...

Reply:
yeah, it's inherited from the superclass where it makes more sense.